### PR TITLE
Feature/book details page cag

### LINKED ISSUE #11: Create Book details component/css file

### PR DESCRIPTION
**Pull request by cagallo**
## What Features are Being Added?

- Added details to render in BookDetails.js
- Added componentDidMount to BookDetails.js 

## What is Being Refactored?

- Deleted extraneous files that we didn't need (React provided)

## Where should the reviewer start?

- BookDetails.js 

## How should this be manually tested?

- go to localhost:3000, click on the 'learn more' button to see single book details 

## What are the relevant tickets?

-  Build BookDetails component 

## Any Other Notes/Questions/Problems?

- None, y'all are great!!!

### CODE REVIEW

File: src/components/BookDetails.js [hunk 1]
```diff
@@ -1,17 +1,42 @@
...
+      .then(bookData => {
+        const cleanedBookData = cleanBookData(bookData[0]);
+        this.setState({ selectedBook: cleanedBookData });
+      })
+      .catch(error => this.setState({ error: error.message }))
+  }
+
+  render() {
+    return (
+        <section className="book-details">
```
Comment (krdurrett) [inline src/components/BookDetails.js:26]: This is a wonderful looking component!

File: src/components/BookDetails.js [hunk 2]
```diff
@@ -1,17 +1,42 @@
...
 
-    render() {
-        return (
-            <h3>details for some stuff</h3>
-        )
-    }
+  componentDidMount = () => {
+    apiCalls.getSingleBook(this.props.id)
+      .then(bookData => {
+        const cleanedBookData = cleanBookData(bookData[0]);
```
Comment (Fordo29) [inline src/components/BookDetails.js:18]: Nice use of calling on a utils file for developer empathy

**Review by krdurrett — APPROVED** [review]
Great job with this component

File: src/components/BookDetails.js [hunk 3]
```diff
@@ -1,17 +1,42 @@
...
+        this.setState({ selectedBook: cleanedBookData });
+      })
+      .catch(error => this.setState({ error: error.message }))
+  }
```
Comment (ectweitmann) [inline src/components/BookDetails.js:22]: Awesome work Chez. This is a nice clean function no pun intended.

**Review by ectweitmann — APPROVED** [review]

**Pull request merged**